ChopChop/Models/Ingredient.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import GRDB
+
+struct Ingredient: Equatable {
+    var id: Int64?
+    var ingredientCategoryId: Int64?
+    var name: String
+    var sets: [Date?: Quantity]
```

Comment:
This will be updated to work with `IngredientBatch`es in the future.

Reply:
For now I wasn't very sure what the rest of you were going to use for your models, so I just used the simplest data structure, which can of course be modified. We just need to remember to update the mapping from the `Record` to the actual model.